salt/states/virt.py

```diff
@@ -367,6 +370,18 @@ def running(name,
 
         .. versionadded:: 3000
 
+    :param serial_type: type of serial connection
+
+        .. versionadded:: 3000
```

Comment:
When adding new parameters now, use `.. versionadded:: Sodium`.
The `Sodium` will be replaced by the proper version number at release time